roles/pulibrary.postgresql/templates/pg_hba.conf.j2

```diff
@@ -0,0 +1,59 @@
+# PostgreSQL Client Authentication Configuration File
+# ===================================================
+#
+
+# DO NOT DISABLE!
+# If you change this first entry you will need to make sure that the
+# database superuser can access the database using some other method.
+# Noninteractive access to all databases is required during automatic
+# maintenance (custom daily cronjobs, replication, and similar tasks).
+#
+# Database administrative login by Unix domain socket
+local   all             postgres                                trust
+# repmgr
+local   replication     {{ repmgr_dbuser_name }}                                  trust
+host    replication     {{ repmgr_dbuser_name }}          {{ replication1 }}      md5
+host    replication     {{ repmgr_dbuser_name }}          {{ replication0 }}      md5
+
+
+local   {{ repmgr_db_name }}     {{ repmgr_dbuser_name }}                                  trust
+host    {{ repmgr_db_name }}     {{ repmgr_dbuser_name }}          {{ replication1 }}      md5
+host    {{ repmgr_db_name }}     {{ repmgr_dbuser_name }}          {{ replication0 }}      md5
+# TYPE  DATABASE        USER            ADDRESS                 METHOD
+
+# "local" is for Unix domain socket connections only
+local   all             all                                     md5
+# IPv4 local connections:
+host    all             all             127.0.0.1/32            md5
+# IPv6 local connections:
+host    all             all             ::1/128                 md5
+# figgy connections
+host    all             all             {{ figgy0 }}        md5
+host    all             all             {{ figgy1 }}       md5
```

Comment:
Can we just do a loop over a variable which is like `postgresql_allowed_hosts` or something, instead of having to edit two files every time we add a host?